python/lsst/ts/m2/csc.py

```diff
@@ -44,20 +45,20 @@
 class M2(salobj.ConfigurableCsc):
     """This is a test CSC for the M2 component with salobj."""
 
+    valid_simulation_modes = (0, 1)
```

Comment:
I know this is anticipating having simulation modes, but you may want to consider adding this when the time comes that we implement it, rather than advertising that it supports simulation when it (still) doesn't.

Reply:
Done in the commit: baf6155. Thanks!